coq.py

```diff
@@ -23,6 +23,7 @@
 import shlex
 import codecs
 import threading
+import sublime
```

Comment:
You are correct that `sublime` has been deliberately left out of this module, for two reasons:
1. Clean abstractions (coq.py is very general and could be re-used in other projects).
2. Testability (coq.py can be tested independently of Sublime---although that independence is [a lot less important](https://github.com/SublimeText/UnitTesting) these days).

In any case, here's how I would structure this change:

- Introduce a new data type in coq.py (perhaps named `CoqResponse`) that contains a list of feedback messages, a list of open goals, and stats about e.g. how many evars there are.
- In the future there can be lots of structure, but for now I'm imagining that each goal in the `CoqResponse` would be represented as a simple string.
- Instead of returning a string, `CoqBot.current_goal` should return a `CoqResponse`.
- `format_response` can be renamed to `parse_response`.
- CoqPlugin.py can have a `def format_response(response : coq.CoqResponse) -> str` that reads the user's current settings and converts the response object to a string.